### DISCUSSION TITLE
Implot support

### DISCUSSION THREAD

**Discussion by ryu-esque** [pkdawson/imgui-godot#48] [Q&A] [answered]
Hey, is there a way to make Implot work with this? Thanks

**Comment by pkdawson** [comment, marked as answer]
In principle anything that works with ImGui should be compatible. But using ImPlot from C# would require bindings, which are not really available in a convenient form.

There's some recent work and discussion about it here:
https://github.com/ImGuiNET/ImGui.NET/issues/435
